### PR TITLE
Update / clarify info on guid types (VTX-2933)

### PR DESCRIPTION
**Pull request by therealsilence**
- Minor reorganization of discussion of guid types
- Clarify guid generation options and best practices
- Other minor tweaks/fixes.

### CODE REVIEW

File: docs/synapse/userguides/storm_ref_type_specific.rstorm [hunk 1]
```diff
@@ -502,52 +505,74 @@ N/A
 Parsing
 +++++++
 
-Guids must be input using their complete 128-bit value. It is generally impractical to manually type
-a guid. Guid forms are most often specified by referencing the node via a more human-friendly secondary
-property. Alternately, the guid value can be copied and pasted.
+It is generally impractical to manually type a guid when lifting or otherwise specifying a guid form. While
+guids can be specfied by a sufficiently unique prefix (e.g., ``ou:org^=4b0c2``), it is more common to reference
```
Comment (Cisphyx) [inline docs/synapse/userguides/storm_ref_type_specific.rstorm:509]:
```suggestion
guids can be specified by a sufficiently unique prefix (e.g., ``ou:org^=4b0c2``), it is more common to reference
```

File: docs/synapse/userguides/storm_ref_type_specific.rstorm [hunk 2]
```diff
@@ -569,173 +593,178 @@ changes (improvements to the ingest logic, additions to the Synapse data model)
...
-- The URL where the article was published (e.g., ``https://www.example.com/my_article.html``)
+**Secondary property deconfliction combined with predictable guid generation** is the most robust method
+for creating guid-based nodes and is recommended in most cases - in particular for any automated data ingest.
+This includes both ongoing ingests (such as via Power-Ups) and one-time scripted ingests (such as via the
+CSV Tool or the Optic Ingest Tool).
 
-Predictable guid values can be generated directly (as part of Storm :ref:`edit-mode` syntax):
+For users who need to create one-off guid forms, this process can be simpilfied via custom Storm commands
+(such as the ``gen.*`` commands) or automation (such as a macro that performs the necessary deconfliction
+and node creation, if a ``gen.*`` command is not avaiilable).
```
Comment (Cisphyx) [inline docs/synapse/userguides/storm_ref_type_specific.rstorm:726]:
```suggestion
and node creation, if a ``gen.*`` command is not available).
```

File: docs/synapse/userguides/storm_ref_type_specific.rstorm [hunk 3]
```diff
@@ -569,173 +593,178 @@ changes (improvements to the ingest logic, additions to the Synapse data model)
...
+Because guid values are unwieldy to use on the command line (outside of copy and paste operations), it
+is often easier to lift guid nodes by a unique secondary property.
 
 **Examples:**
 
 Lift an org node by its alias:
 
 .. storm-pre:: [ ou:org=* :alias=choam :name='combine honnete ober advancer mercantiles' ]
-.. storm-cli:: ou:org:alias=choam
+.. storm-cli:: ou:org:alias = choam
```
Comment (Cisphyx) [inline docs/synapse/userguides/storm_ref_type_specific.rstorm:762]: Is "spaces around operators" the style we want to move towards? We definitely have a mix of both ways but it seems like currently we have more instances without spaces around the operators.

Comment (therealsilence) [reply docs/synapse/userguides/storm_ref_type_specific.rstorm:762]: I don't think we have a strong preference - I've been trying to include spaces recently b/c I think it helps the readability but I don't feel strongly enough about it to go update the entire file (though the type-A in me was tempted). :wink:

Comment (Cisphyx) [reply docs/synapse/userguides/storm_ref_type_specific.rstorm:762]: Yeah they do seem a bit more readable especially with the operators that are more than just `=`, and updating everything does seem a bit much to do all at once.

**Review by Cisphyx — APPROVED** [review]

**Pull request merged**